server/schema/squiggle/squiggle.resolver.js

```diff
@@ -33,6 +33,32 @@ module.exports = {
       throw APIError(null, error);
     }
   },
+
+  updateSquiggle: async (
+    _parent,
+    { id, newContent },
+    { session, authToken, decodedToken, API: { Squiggle } }
+  ) => {
+    try {
+      if (!UserPermission.exists(session, authToken, decodedToken, 'squiggle.write.new')) {
```

Comment:
According to [this](https://github.com/Monday-Morning/project-reclamation/blob/main/planning/v1/PERMISSIONS.md#squiggle-section) it should be squiggle.write.all.